mod/item.php

```diff
@@ -842,7 +844,7 @@ function item_post(App $a) {
 		file_tag_update_pconfig($uid,$categories_old,$categories_new,'category');
 
 		Worker::add(PRIORITY_HIGH, "Notifier", 'edit_post', $post_id);
-		if ((x($_REQUEST, 'return')) && strlen($return_path)) {
+		if (x($_REQUEST, 'return') && strlen($return_path)) {
```

Comment:
`$return_path` already is the value of `$_REQUEST['return']` if it exists, so this condition can be simplified to `if ($return_path) {...}`